test_env.py

```diff
@@ -58,26 +32,10 @@ def setup_class(cls):
         """
         result = kube_env.setup_bookinfo_deployment("MK", False)
         assert result == util.EXIT_SUCCESS
-        # start the modified bookinfo app, ignore the result for now
-        result = kube_env.install_modded_bookinfo()
 
-    @pytest.mark.parametrize("query_file,query_udfs", KUBE_QUERIES, ids=KUBE_IDS)
-    def test_deployment(self, query_file, query_udfs):
-        # generate the filter code
-        cmd = f"{COMPILER_BINARY} "
-        cmd += f"-q {QUERY_DIR.joinpath(query_file)} "
-        for query_udf in query_udfs:
-            udf_file = UDF_DIR.joinpath(query_udf)
-            udf_file = udf_file.with_suffix(".cc")
-            cmd += f"-u {udf_file} "
-        cmd += "-r productpage-v1 "
-        result = util.exec_process(cmd)
-        assert result == util.EXIT_SUCCESS
-        # build the filter
-        result = kube_env.build_filter(kube_env.FILTER_DIR)
-        assert result == util.EXIT_SUCCESS
-        # deploy the filter
-        result = kube_env.refresh_filter(kube_env.FILTER_DIR)
+
```

Comment:
Why not keep the parameterized structure and just xfail them while the compiler is broken?

Reply:
Turns out the queries for the tests are a little to specific so it does not really make sense to keep them right now. The idea is to replace them with a structure like `test_count`.

Reply:
Why are they too specific?  I think test_count could be parameterized with just the query and result file names, and a list of expected results written to storage, couldn't it?

Reply:
Every query covers a slightly different use case. `count` we send a request three times and check that the increment works correctly. `breadth` we only send once since that information does not change. Or it may change depending on the header we provide (meaning we route a different path). `response_code_count` we will have to introduce a failure and so on. I think the queries are just too different for a unified approach, as I initially thought.

Reply:
Okay, that makes sense.